### PR TITLE
Module commit

### PR DESCRIPTION
**Pull request by tlipatov**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+provider "aws" {
+  version = "~> 1.40"
+  region = "${var.aws_region}"
+}
+
+data "aws_secretsmanager_secret_version" "this" {
```
Comment (eugenejen) [inline main.tf:6]: i don't think this is specific enough

Comment (eugenejen) [reply main.tf:6]: or you just want a general place holder?

**Review by eugenejen — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (tlipatov) [reply main.tf:6]: Its a reusable module, "this" is only for internal reference.
Using the term "this" seems to be adapted by the community as as standard way to reference module internals

Comment (eugenejen) [reply main.tf:6]: ok. if this convention. then no problem. we probably need to find a terraform style guide link as reference

**Review by eugenejen — APPROVED** [review]

**Comment by jcarr-sailthru** [comment]
Nice one.

**Pull request merged**